### PR TITLE
Feature/lil/get drive hisotry days

### PR DESCRIPTION
**Pull request by JangHyeonJun2**
# 해결하려는 문제가 무엇인가요?
* 택시 드라이버의 일일 운행량 조회를 위한 전초 작업..

# 어떻게 해결했나요?
* 일단 택시 조회 API가 필요하기 때문에 택시조회 API 생성
* 택시 드라이버 테스트 코드가 없어서 틀 생성

## Attachment

### CODE REVIEW

**Comment by TonyKim9401** [comment]
### 📝 테스트 커버리지 리포트입니다
|File|Coverage [95.8%]|:green_apple:|
|:-|:-:|:-:|
|[TaxiDriverController.kt](https://github.com/KATJ-HH2/katj/blob/5d54d58dee13b7c72b11d6df9196de570ebc5505/src%2Fmain%2Fkotlin%2Fcom%2Fhh2%2Fkatj%2Ftaxidriver%2Fcontroller%2FTaxiDriverController.kt)|100%|:green_apple:|
|[TaxiReader.kt](https://github.com/KATJ-HH2/katj/blob/5d54d58dee13b7c72b11d6df9196de570ebc5505/src%2Fmain%2Fkotlin%2Fcom%2Fhh2%2Fkatj%2Ftaxi%2Fcomponent%2FTaxiReader.kt)|100%|:green_apple:|
|[TaxiController.kt](https://github.com/KATJ-HH2/katj/blob/5d54d58dee13b7c72b11d6df9196de570ebc5505/src%2Fmain%2Fkotlin%2Fcom%2Fhh2%2Fkatj%2Ftaxi%2Fcontroller%2FTaxiController.kt)|100%|:green_apple:|
|[FindTaxiResponse.kt](https://github.com/KATJ-HH2/katj/blob/5d54d58dee13b7c72b11d6df9196de570ebc5505/src%2Fmain%2Fkotlin%2Fcom%2Fhh2%2Fkatj%2Ftaxi%2Fmodel%2Fdto%2Fresponse%2FFindTaxiResponse.kt)|100%|:green_apple:|
|[TaxiService.kt](https://github.com/KATJ-HH2/katj/blob/5d54d58dee13b7c72b11d6df9196de570ebc5505/src%2Fmain%2Fkotlin%2Fcom%2Fhh2%2Fkatj%2Ftaxi%2Fservice%2FTaxiService.kt)|87.5%|:green_apple:|

|Total Project Coverage|69.38%|:green_apple:|
|:-|:-:|:-:|

File: src/main/kotlin/com/hh2/katj/taxi/model/dto/response/FindTaxiResponse.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.hh2.katj.taxi.model.dto.response
+
+import com.hh2.katj.taxi.model.FuelType
+
+data class FindTaxiResponse(
+    val id: Long,
+    val carNo: String,
+    val fuelType: FuelType,
+    val color: String
+)
```
Comment (TonyKim9401) [inline src/main/kotlin/com/hh2/katj/taxi/model/dto/response/FindTaxiResponse.kt:10]: 지난번 하헌우 코치님 조언대로 Response가 중점이 되도록 네이밍 괜찮지 않을까 생각됩니다.
이름 짓기가 참 어렵네요 ㅎㅎ

**Review by TonyKim9401 — APPROVED** [review]
고생하셨습니다~!

File: src/test/kotlin/com/hh2/katj/taxi/TaxiAcceptanceTest.kt [hunk 2]
```diff
@@ -10,7 +11,9 @@ import org.assertj.core.api.Assertions
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.DisplayName
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.function.Executable
 import java.time.LocalDate
+import org.junit.jupiter.api.Assertions as JupiterApiAssertions
```
Comment (jiseongTak) [inline src/test/kotlin/com/hh2/katj/taxi/TaxiAcceptanceTest.kt:16]: 이런식으로도 사용할 수 있군요..

File: src/main/kotlin/com/hh2/katj/taxi/model/dto/response/FindTaxiResponse.kt (hunk 1 above)
Comment (JangHyeonJun2) [reply src/main/kotlin/com/hh2/katj/taxi/model/dto/response/FindTaxiResponse.kt:10]: 엇.. Response가 메인으로 이름으로 작성해봤는데,, 조금 더 고민해볼게요!

**Comment by TonyKim9401** [comment]
### 📝 테스트 커버리지 리포트입니다
|File|Coverage [75.26%]|:x:|
|:-|:-:|:-:|
|[TaxiReader.kt](https://github.com/KATJ-HH2/katj/blob/4fd79e5e80a7b765ebab980a017e96104bdc1db0/src%2Fmain%2Fkotlin%2Fcom%2Fhh2%2Fkatj%2Ftaxi%2Fcomponent%2FTaxiReader.kt)|100%|:green_apple:|
|[TaxiController.kt](https://github.com/KATJ-HH2/katj/blob/4fd79e5e80a7b765ebab980a017e96104bdc1db0/src%2Fmain%2Fkotlin%2Fcom%2Fhh2%2Fkatj%2Ftaxi%2Fcontroller%2FTaxiController.kt)|100%|:green_apple:|
|[FindTaxiResponse.kt](https://github.com/KATJ-HH2/katj/blob/4fd79e5e80a7b765ebab980a017e96104bdc1db0/src%2Fmain%2Fkotlin%2Fcom%2Fhh2%2Fkatj%2Ftaxi%2Fmodel%2Fdto%2Fresponse%2FFindTaxiResponse.kt)|100%|:green_apple:|
|[TaxiService.kt](https://github.com/KATJ-HH2/katj/blob/4fd79e5e80a7b765ebab980a017e96104bdc1db0/src%2Fmain%2Fkotlin%2Fcom%2Fhh2%2Fkatj%2Ftaxi%2Fservice%2FTaxiService.kt)|87.5%|:green_apple:|
|[TaxiDriverController.kt](https://github.com/KATJ-HH2/katj/blob/4fd79e5e80a7b765ebab980a017e96104bdc1db0/src%2Fmain%2Fkotlin%2Fcom%2Fhh2%2Fkatj%2Ftaxidriver%2Fcontroller%2FTaxiDriverController.kt)|18%|:x:|

|Total Project Coverage|62.97%|:green_apple:|
|:-|:-:|:-:|

**Pull request merged**